packages/hmr/reactivity/hmr/proxy.py

```
@@ -0,0 +1,19 @@
+from collections.abc import MutableMapping
+from typing import Any
+
+from ..context import Context
+from ..helpers import Reactive
+
+
+class Proxy[T: MutableMapping](Reactive[str, Any]):
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (reactivity.hmr.proxy, line 8)'

(E0001)